services-turf/src/main/java/com/mapbox/turf/TurfMeasurement.java

```diff
@@ -439,4 +439,40 @@ public static Polygon bboxPolygon(@NonNull BoundingBox boundingBox) {
                 Point.fromLngLat(boundingBox.west(), boundingBox.north()),
                 Point.fromLngLat(boundingBox.west(), boundingBox.south()))));
   }
+
+
+  /**
+   * Takes a bounding box and calculates the minimum square bounding box that would contain the input.
+   *
+   * @param boundingBox extent in west, south, east, north order
+   * @return a square surrounding bbox
+   */
+  public static BoundingBox square(@NonNull BoundingBox boundingBox) {
+    double horizontalDistance = distance(
+            Point.fromLngLat(boundingBox.west(), boundingBox.south()),
+            Point.fromLngLat(boundingBox.east(), boundingBox.south())
+    );
+    double verticalDistance = distance(
+            Point.fromLngLat(boundingBox.west(), boundingBox.south()),
+            Point.fromLngLat(boundingBox.west(), boundingBox.north())
```

Comment:
you can just use boundingBox.northeast() instead of creating a point

Reply:
there is no northEast coordinate used